src/views/preview/comment/compose-comment.jsx

```diff
@@ -80,20 +81,23 @@ class ComposeComment extends React.Component {
                 let muteOpen = false;
                 let muteExpiresAtMs = 0;
                 let rejectedStatus = ComposeStatus.REJECTED;
+                let showWarning = false;
                 if (body.status && body.status.mute_status) {
                     muteExpiresAtMs = body.status.mute_status.muteExpiresAt * 1000; // convert to ms
                     rejectedStatus = ComposeStatus.REJECTED_MUTE;
                     if (this.shouldShowMuteModal(body.status.mute_status.offenses)) {
```

Comment:
In general we're not showing the full modal if you get muted multiple times within a 7-day period. However, if you get the warning we should probably show the modal again.
```suggestion
                    if (body.status.mute_status.showWarning || this.shouldShowMuteModal(body.status.mute_status.offenses)) {
```

Reply:
Sounds good. I'll build this into "shouldShowMuteModal" so all the logic about whether to show it lives (and is tested) through that.